packages/compartment-mapper/src/bundle-json.js

```diff
@@ -0,0 +1,23 @@
+/* Provides JSON support for `bundle.js`. */
+
+const textDecoder = new TextDecoder();
+
+export default {
+  runtime: '',
+  getBundlerKit({ index, indexedImports, bytes }) {
+    const json = textDecoder.decode(bytes).trim();
```

Comment:
(could) decode and encode back, in case it's not actually JSON

Reply:
I’ve followed-up with a change that aligns with this reasoning. I hesitated because this eliminates whatever spacing choices the author had made.